nilearn/maskers/multi_nifti_maps_masker.py

```diff
@@ -137,6 +143,7 @@ def __init__(
         t_r=None,
         dtype=None,
         resampling_target="data",
+        keep_masked_maps=True,
```

Comment:
change in API but this parameter exists in the parent class
so to me this is a bug fix more than an enhancement because users could have done